src/main/java/no/nav/familie/integrasjoner/dokarkiv/DokarkivService.kt

```diff
@@ -41,14 +46,28 @@ class DokarkivService(private val dokarkivRestClient: DokarkivRestClient,
 
         val metadata = dokarkivMetadata.getMetadata(arkiverDokumentRequest.hoveddokumentvarianter[0])
         val hoveddokument = mapHoveddokument(arkiverDokumentRequest.hoveddokumentvarianter)
-        val vedleggsdokumenter = arkiverDokumentRequest.vedleggsdokumenter.map(this::mapTilArkivdokument)
+        var vedleggsdokumenter = arkiverDokumentRequest.vedleggsdokumenter.map(this::mapTilArkivdokument)
         val jpsak: Sak? = if (arkiverDokumentRequest.fagsakId != null)
             Sak(fagsakId = arkiverDokumentRequest.fagsakId,
                 sakstype = "FAGSAK",
                 fagsaksystem = metadata.fagsakSystem) else null
 
         val navn = hentNavnForFnr(fnr = arkiverDokumentRequest.fnr, behandlingstema = metadata.tema)
 
+        //førsteside
+        val førsteside: ByteArray? = if (arkiverDokumentRequest.førsteside != null)
+            førstesideGeneratorService.genererForside(arkiverDokumentRequest.førsteside!!) else null
```

Comment:
Må man ha !! her, etter null-sjekken?

Reply:
ja, måtte det. Hvis ikke klager intellij på "cross-module smart casts are not allowed."